docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+services:
+  hydra:
+    container_name: hydra
+    image: ghcr.io/hydrasco/hydra:latest
```

Comment:
@wuputah heads up that using `latest` probably won't work how you expect.  This will never pull another "newer" local, so once you've got any `ghcr.io/hydrasco/hydra:latest` on your system it won't pull another.